modules/library/documents/library-document.php

```diff
@@ -44,6 +44,10 @@ public static function get_properties() {
 		$properties['admin_tab_group'] = 'library';
 		$properties['show_in_library'] = true;
 		$properties['register_type'] = true;
+		$properties['cpt'] = [ Source_Local::CPT ];
+
+		// TODO: Remove - Backwards compatibility, since 'Theme_Document' does not support 'static::get_type'.
+		$properties['show_in_finder'] = false;
```

Comment:
Is something will not work now?

Reply:
Yes, `Theme_Document` calls eventually to 'get_site_editor_type_bc'. And produce error.

Reply:
Please create a PR the fix it and link it here.